dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/postgres/V7.3_2022.04.29__orcid_queue_and_history.sql

```diff
@@ -0,0 +1,54 @@
+--
+-- The contents of this file are subject to the license and copyright
+-- detailed in the LICENSE and NOTICE files at the root of the source
+-- tree and available online at
+--
+-- http://www.dspace.org/license/
+--
+
+-----------------------------------------------------------------------------------
+-- Create tables for ORCID Queue and History
+-----------------------------------------------------------------------------------
+
+CREATE SEQUENCE orcid_queue_id_seq;
+
+CREATE TABLE orcid_queue
+(
+    id INTEGER NOT NULL,
+    owner_id uuid NOT NULL,
+    entity_id uuid,
+    attempts INTEGER,
+    put_code CHARACTER VARYING(255),    
+	record_type CHARACTER VARYING(255),     
+	description CHARACTER VARYING(255),     
+	operation CHARACTER VARYING(255),      
+	metadata TEXT,                    
```

Comment:
Small note... the alignment of these SQL files is messy (in all the files)... it looks like in some places tabs are used when it should be spaces.  It'd be good to clean this up now as Flyway does a checksum of these files to validate...so if we wait to clean this up later, Flyway could throw validation errors.